sound/soc/sof/topology.c

```diff
@@ -3359,7 +3359,7 @@ static int sof_complete(struct snd_soc_component *scomp)
 			 * Apply the dynamic_pipeline_widget flag and set the pipe_widget field
 			 * for all widgets that have the same pipeline ID as the scheduler widget
 			 */
-			list_for_each_entry_reverse(comp_swidget, &sdev->widget_list, list)
+			list_for_each_entry(comp_swidget, &sdev->widget_list, list)
```

Comment:
Im good with this change but there's no change in logic whether we walk in reverse or not for this one.

Reply:
so might as well not confuse everyone.